sdk/eventhub/event-hubs/test/client.spec.ts

```diff
@@ -10,304 +10,374 @@ import chaiString from "chai-string";
+    try {
+      await client.getPartitionProperties("0");
+    } catch (err) {
+      errorGetPartitionProperties = err.message;
+    }
+    should.equal(errorGetPartitionProperties, expectedErrorMsg, "Expected error not thrown for getPartitionProperties()");
+
+    let errorGetProperties: string = "";
+    try {
+      await client.getProperties();
+    } catch (err) {
+      errorGetProperties = err.message;
+    }
+    should.equal(errorGetProperties, expectedErrorMsg, "Expected error not thrown for getProperties()");
+
   });
-}).timeout(60000);
-
-// describe("Test AadTokenCredentials", function(): void {
-//   let errorWasThrown: boolean = false;
-//   before("validate environment", function(): void {
-//     should.exist(
-//       env[EnvVarKeys.AZURE_CLIENT_ID],
-//       "define AZURE_CLIENT_ID in your environment before running integration tests."
-//     );
-//     should.exist(
-//       env[EnvVarKeys.AZURE_TENANT_ID],
-//       "define AZURE_TENANT_ID in your environment before running integration tests."
-//     );
-//     should.exist(
-//       env[EnvVarKeys.AZURE_CLIENT_SECRET],
-//       "define AZURE_CLIENT_SECRET in your environment before running integration tests."
-//     );
-//     should.exist(env[EnvVarKeys.ENDPOINT], "define ENDPOINT in your environment before running integration tests.");
-//   });
-
-//   async function testAadTokenCredentials(client: EventHubClient): Promise<void> {
-//     const sender = client.createProducer();
-//     const partitionIds = await client.getPartitionIds();
-//     const receiver = await client.createConsumer(partitionIds[0]);
-//     await sender.send({ body: "Hello world" });
-//     const msgs = await receiver.receiveBatch(1);
-
-//     should.equal(msgs.length, 1, "Unexpected number of messages");
-//   }
-
-//   it("throws error for invalid tokenCredentials", async function(): Promise<void> {
-//     const client = new EventHubClient(env.ENDPOINT, env.EVENTHUB_NAME);
-//     await testAadTokenCredentials(client).catch((err: any) => {
-//       errorWasThrown = true;
-//       should.equal(
-//         err.message,
-//         "Please provide a token credentials interface or a valid object of SharedKeyCredential."
-//       );
-//     });
-//     should.equal(errorWasThrown, true, "Error thrown flag must be true");
-//   });
-
-//   it("sends a message to the Event Hub entity", async function(): Promise<void> {
-//     const credential = new EnvironmentCredential();
-//     const client = new EventHubClient(env.ENDPOINT, env.EVENTHUB_NAME, credential);
-//     await testAadTokenCredentials(client);
-//     await client.close();
-//   });
-// });
+
+  it("errors after close() on sender", async function(): Promise<void> {
+    const senderErrorMsg =
+      `The sender for "${client.eventHubName}" has been closed and can no longer be used. ` +
+      `Please create a new sender using the "createProducer" function on the EventHubClient.`;
```

Comment:
Should we update the error messages to use consumer/producer instead of receiver/sender? I didn't go over the error messages when updating the JSDoc comments.

Reply:
Yes, we probably should. Can you create an issue for it?

Reply:
Created #3994